kmsencryption/lib.py

```diff
@@ -53,19 +55,36 @@ def decrypt(data, env, profile, prefix):
     return decrypt_value(data, prefix, kms_key_provider)
 
 
+def decrypt_object(input, prefix, kms_key_provider):
```

Comment:
Minor: I'd not recommend using `input` as a variable name, as it shadows the python builtin. This could be potentially error prone or make IDE crazy. Of course *it works and is correct*, so I'm leaving it to you, whether to avoid it or not.

Reply:
Trivial: In the funcs above the `kms_key_provider` was called `key_provider`, let unify that